tests/contrib/patch.py

```diff
@@ -354,9 +375,7 @@ def test_unpatch_idempotent(self):
                 self.assert_not_wrapped(redis.client.BasePipeline.execute)
                 self.assert_not_wrapped(redis.client.BasePipeline.immediate_execute_command)
             """
-            for module_name, integration_name, unpatch in self.MODULES:
-                self.assert_module_not_imported(module_name)
-                unpatch()
-                unpatch()
-                module = importlib.import_module(module_name)
-                self.assert_not_patched(module)
+            self.assert_module_not_imported(self.__module_name__)
+            self.__unpatch_func__()
```

Comment:
we are only unpatching once here.

Also, should we patch first, unpatch to assert it is unpatched, and then unpatch again to assert it is unpatched?